python/lsst/pipe/tasks/make_direct_warp.py

```diff
@@ -316,6 +336,36 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
 
         visit_summary = inputs["visit_summary"] if self.config.useVisitSummaryPsf else None
 
+        # Can this go in the run method?
+        if self.config.doSelectPreWarp:
+            dataIdList = [ref.datasetRef.dataId for ref in inputRefs.calexp_list]
+            bboxList, wcsList = [], []
+            for dataId in dataIdList:
+                row = visit_summary.find(dataId["detector"])
+                if row is None:
+                    raise RuntimeError(f"Unexpectedly incomplete visit_summary: {dataId=} is missing.")
+                bboxList.append(row.getBBox())
+                wcsList.append(row.getWcs())
+
+            # inputs["bboxList"] = bboxList
+            # inputs["wcsList"] = wcsList
```

Comment:
Should these be uncommented now?  Or deleted?

Reply:
Deleted them. I'm not a big fan of shoving more items in the `inputs` dictionary when we can pass them explicitly.